### DISCUSSION TITLE
[A-L] (2024/25) Foglio 1 - Esercizio 10

### DISCUSSION THREAD

**Discussion by Rurik-D** [sapienzastudentsnetwork/algebra#471] [Esercizi - Viaggi/Cherubini]
![image](https://github.com/user-attachments/assets/3e45eb95-f385-4414-ae81-c19d8620789b)

**Comment by IonutCicio** [comment]
È possibile trovare una mia soluzione in uno degli esercizi di [Calcolo delle Probabilità](https://github.com/sapienzastudentsnetwork/calcolo-delle-probabilita/discussions/153#discussioncomment-8247023).

Sebbene la terza richiesta possa sembrare semplice, in realtà richiede un ragionamento combinatorio relativamente complesso (come si può notare in questa discussione di [Metodi Matematici per l'Informatica](https://github.com/sapienzastudentsnetwork/metodi-matematici-per-l-informatica/discussions/248))
